{
    "type":"server_log",
    "packages":{
        "rich":"12.4.4",
        "typing_extensions":"4.2.0",
        "Pygments":"2.12.0",
        "altgraph":"0.17",
        "autopep8":"1.5.7",
        "click":"8.1.2",
        "dnspython":"1.16.0",
        "Flask":"2.0.3",
        "Flask-WTF":"0.15.1",
        "gunicorn":"20.1.0",
        "importlib-metadata":"4.11.3",
        "itsdangerous":"2.1.2",
        "Jinja2":"3.1.1",
        "macholib":"1.14",
        "MarkupSafe":"2.1.1",
        "numpy":"1.22.3",
        "openpyxl":"3.0.9",
        "pandas":"1.4.2",
        "pip":"22.1.2",
        "pycodestyle":"2.7.0",
        "pyinstaller":"4.2",
        "pymongo":"3.12.0",
        "pytz":"2022.1",
        "setuptools":"49.2.1",
        "six":"1.15.0",
        "toml":"0.10.2",
        "Werkzeug":"2.1.1",
        "wheel":"0.36.2",
        "WTForms":"2.3.3",
        "zipp":"3.8.0"
    },
    "info":[
        "Package managing version 3",
        "Connectting TCP/IP",
        "Redirect to pip pool service for advanced version downloads",
        "Connection established successfully",
        "Searching for dependencies",
        "Installation Started"
    ],
    "finish":[
        "<name> Collection downloads successfully ---><<version>> is stabled"
    ]
}